std/meta.d

```diff
@@ -1145,6 +1145,13 @@ unittest
 
     static assert(is(Filter!(ApplyRight!(isImplicitlyConvertible, short),
         ubyte, string, short, float, int) == AliasSeq!(ubyte, short)));
+
+    static assert(is(typeof({
```

Comment:
Agreed. Id -> Alias in meta.d? I tried the new implementation used Alias and all tests passed.